stable/azuredisk-csi-driver/values.yaml

```diff
@@ -4,31 +4,31 @@ image:
     tag: v0.5.0
     pullPolicy: Always
   csiProvisioner:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-provisioner
+    repository: quay.io/k8scsi/csi-provisioner
     tag: v1.4.0
     pullPolicy: Always
   csiAttacher:
-    repository: mcr.microsoft.com/oss/kubernetes-csi/csi-attacher
+    repository: quay.io/k8scsi/csi-attacher
     tag: v1.2.0
```

Comment:
v2.1.1 got released, but maybe something for the next change

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade

Reply:
I am aware is using this specific versions, but that is what we believe it is being tested so... that is what we should use

Reply:
> @sebbrandt87 I suggest you next time to follow this repository to perform any upgrade https://github.com/kubernetes-sigs/azuredisk-csi-driver.

well actually I decline, especially if I see bugs popping up in kubernetes version related to the `k8scsi` released images that are fixed in new versions and still api compatible to csi